Part-1/Exercise-1.ipynb

```diff
@@ -6,7 +6,17 @@
     "id": "yWGTIPMXotxs"
    },
    "source": [
-    "# **Exercise 1. Positive logic programs.**\n"
+    "# **Exercise 1. Positive logic programs.**\n",
```

Comment:
Hi Etienne, 

Dont worry for this time, but for the next time, before doing a pull request, please pull the latest version of the branch.
I had deleted the instructions about the exercises.

Chao!
javier